repo/fsrepo/migrations/ipfsfetcher_test.go

```diff
@@ -0,0 +1,47 @@
+package migrations
+
+import (
+	"bufio"
+	"context"
+	"testing"
+)
+
+func TestIpfsFetcher(t *testing.T) {
+	t.Skip("manually-run dev test only")
```

Comment:
maybe hide it behind a flag, then :) otherwise the only way to run it is by editing the source.

is there any way to eventually make this test work in a local way without extra setup? e.g. by setting up some IPFS/HTTP servers/nodes.

Reply:
Done - hidden behind flag

For the HttpFetcher I created a dummy test server.  I am not really sure there is a great way to do this for IPFS.  Mocking interfaces seems a bit excessive.

Reply:
I'm just a bit worried that we have a test that will practically never be run. Mocking isn't great, but if running a real IPFS node for the test is too difficult, at least testing with a mock by default would be better than testing nothing by default. The flag could still be used to test against a real IPFS node instead of a mock.

Reply:
Another pattern is to use an environment variable and put the name of it in the Skip message. Inspired by this blog post https://peter.bourgon.org/blog/2021/04/02/dont-use-build-tags-for-integration-tests.html

Reply:
This was handled by adding a `skipUnlessEpic` function like as is done here:
 https://github.com/ipfs/go-ipfs/blob/ef866a1400b3b2861e5e8b6cc9edc8633b890a0a/test/integration/addcat_test.go#L173?